content/hi/docs/reference/glossary/application-architect.md

```diff
@@ -0,0 +1,18 @@
+---
+title: एप्लीकेशन आर्किटेक्ट
+id: application-architect
+date: 2018-04-12
+full_link:
+short_description: >
+  किसी एप्लिकेशन के उच्च-स्तरीय डिज़ाइन के लिए जिम्मेदार व्यक्ति।
```

Comment:
Wouldn't रचना be better than the literal translation of design?

Reply:
Yes Actually I was thinking to use रचना but anyone might get confused with design
so I did not use it
Nice suggestion 👍 


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient